moped-editor/src/views/projects/projectView/ProjectNotes/NoteTypeButton.js

```diff
@@ -15,15 +17,24 @@ const NoteTypeButton = ({
   setFilterNoteType,
   filterNoteType,
   label,
-}) => (
-  <Button
-    color="primary"
-    sx={{ margin: 2 }}
-    variant={filterNoteType === noteTypeId ? "contained" : "outlined"}
-    onClick={() => setFilterNoteType(noteTypeId)}
-  >
-    {label}
-  </Button>
-);
+  isDisabled,
+  disabledMessage,
+}) => {
+  return (
+    <Tooltip title={isDisabled ? disabledMessage : ""}>
+      <span>
```

Comment:
This was a TIL for me that [tooltips won't show on disabled buttons since user interactions aren't trigger on them](https://v5.mui.com/material-ui/react-tooltip/#disabled-elements). Could you add a comment or link to the docs here to help us avoid an accidental regression? 🙏

Reply:
+1, I could definitely see future me deleting this span that is silently doing so much work here